src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -793,3 +797,198 @@ def plot_GP_components(lc, model, meta, fitter, isTitle=True):
         fig.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
         if not meta.hide_plots:
             plt.pause(0.2)
+
+
+def plot_fleck_star(lc, model, meta, fitter):
+    
+    # find number of spots
+    spotinds = [s for s in model.parameters.dict.keys() if 'spotrad' in s]   
+    counter = [s for s in spotinds if '_' in s]
+    nspots = len(spotinds) - len(counter)
+
+    # find number of limb darkening parametesr
+    limbdarks = [s for s in model.parameters.dict.keys() if 'u' in s] 
+    counter = [s for s in limbdarks if '_' in s]
+    nus = len(limbdarks) - len(counter)
+    
+    spotrad = np.zeros((lc.nchannel_fitted, nspots))
+    spotlat = np.zeros((lc.nchannel_fitted, nspots))
+    spotlon = np.zeros((lc.nchannel_fitted, nspots))
+    spotcon = np.zeros((lc.nchannel_fitted))
+    starrot = np.zeros((lc.nchannel_fitted))
+    starinc = np.zeros((lc.nchannel_fitted))
+
+    uarray = np.zeros((lc.nchannel_fitted, nus))
+
+    # Set spot/fleck parameters
+    # set a star rotation for the star object
+    # not actually used in fast mode. 
+    # overwritten if user supplies and runs slow mode
+    starrot[:] = 100 
+
+    pid_id = ''
```

Comment:
No need to make this empty string if it is never updated below (and none of the plots depend on the planet number)